src/config/networks.ts

```diff
@@ -65,11 +65,13 @@ if (import.meta.env.REACT_APP_DISABLE_MAINNET !== '1') {
       yearlyInflationInTokens: BN_MILLION.mul(new BN(30)).toNumber(),
     },
     defaultFeeReserve: 0.1,
-  };
+  } as const;
+
+  NetworkList[alephZero.name] = alephZero;
```

Comment:
I think 
```ts
NetworkList[alephZero.name] = { ... } as const
```
should still work?

Reply:
riddle me this - what is `alephZero` :sweat_smile:

Reply:
Nvm, all good!